libraries/ms-common/src/stm32f0xx/gpio.c

```diff
@@ -3,6 +3,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "gpio_cfg.h"
 #include "status.h"
 #include "stm32f0xx.h"
 #include "stm32f0xx_gpio.h"
```

Comment:
I mean, stm32f0xx.h includes _gpio.h as well, so it's redundant. I understand that it's nice to be explicit, but I think it can get messy pretty easily (see: interrupt.c)